src/test/resources/application-integration-test.yml

```diff
@@ -1,13 +1,24 @@
 workspace.integration-test:
   data-repo-instance-names:
+    local: terra
     dev: terra
+    alpha: terra
+    staging: terra
   data-repo-snapshot-id:
+    local: 97b5559a-2f8f-4df3-89ae-5a249173ee0c
     dev: 97b5559a-2f8f-4df3-89ae-5a249173ee0c
-  user-email: william.thunderlord@test.firecloud.org
+    alpha: d56f4db5-b6c6-4a7e-8be2-ff6aa21c4fa6
+    staging:
```

Comment:
if we don't know the snapshot id, can we have the test fail? Maybe leaving the `staging:` key out entirely would cause a failure ... or, lmk if it already fails with the code as-is

Reply:
added a placeholder value